src/main/java/svc/controllers/CourtController.java

```diff
@@ -26,8 +27,8 @@
 	}
 	
 	@GetMapping(value = "courts/{id}")
-	public Court getOne(@PathVariable final Long id){
-		return courtRepository.findOne(id);
+	public Optional<Court> getOne(@PathVariable final Long id){
```

Comment:
How does returning an optional work here? A quick google search yielded me no results in Spring what this does. Hopefully a 404 when there is no Court?